README.md

```diff
@@ -109,41 +111,62 @@ app.run()                     # opens a window and starts the game.
 
    To close the window, you can by default, press shift+q or press the red x. to disable this, write `window.exit_button.enabled = False` somewhere in your code.
 
+## Documentation
+If you need help, you can watch documentation sites:
+   * [Ursina Documentation](https://www.ursinaengine.org/documentation.html)
+      - Can give you a lot of help, because it explains how Ursina works
+   * [Ursina API Reference](https://www.ursinaengine.org/api_reference.html)
+      - List the different functionalities of ursina, with example code
+   * [Samples](https://www.ursinaengine.org/samples.html)
+      - Little games with simple code, as examples
+
+You found a bug? You can help ursina by posting it in [the issues](https://github.com/pokepetter/ursina/issues), or by making a [pull request](https://github.com/pokepetter/ursina/pulls)
+
 
-## Project Structure
-```
 ## Project Structure
 
-📁docs
-    📃index.txt
-    📃documentation.txt
-    📃inventory_tutorial.txt
-    ...
-        # text files for the website. gets turned into .html files with sswg.
-
-    📃cheat_sheet.html
-        # auto generated documentation made with documentation_generator.py.
-
-    📃tutorial_generator.py
-        # turns specific .py files into .txt files, which can then be turned into .html by sswg.
-        # this extracts the comments from the source files into description for that step and the code after into code blocks.
-        # see platformer_tutorial.py for an example.
-
-📁samples               # small example games.
-
-📁ursina                # the actual ursina module.
-    📁audio                 # built-in audio clips.
-    📁editor                # the 3d level editor for ursina.
-    📁fonts                 # built-in fonts.
-    📁models                # .blend files, source files, for built-in 3d models.
-        📁procedural            # classes for generating 3d models, like Cylinder, Quad and Terrain.
-    📁models_compressed     # .blend files converted to .ursinamesh.
-    📁prefabs               # higher level classes like Draggable, Slider, Sprite, etc.
-
-    📃__init__.py
-    📃application.py
-    📃audio.py
-    ...
-        # ursina base modules, like code for Entity, input_handler, Text, window and so on.
+- 📁docs

```

Comment:
This part looks worse than before in my option. The extra newlines and added styling makes it harder to read.

Reply:
Reverted it, now it is like before